kolena/_experimental/object_detection/evaluator_multiclass.py

```diff
@@ -168,11 +167,7 @@ def compute_test_sample_metrics(
         assert configuration is not None, "must specify configuration"
         # compute thresholds to cache values for subsequent steps
         self.compute_and_cache_f1_optimal_thresholds(configuration, inferences)
-        labels = {gt.label for _, gts, _ in inferences for gt in gts.bboxes}
-        return [
-            (ts, self.compute_image_metrics(gt, inf, configuration, test_case.name, labels))
-            for ts, gt, inf in inferences
-        ]
+        return [(ts, self.compute_image_metrics(gt, inf, configuration, test_case.name)) for ts, gt, inf in inferences]
```

Comment:
What's the reason for dropping the `labels` here?

Reply:
Discussed internally - to ignore nothing from the inferences